pkg/controller/janitor/janitor_controller.go

```diff
@@ -58,7 +52,7 @@ func NewController(
 
 	controller := &Controller{
 		recorder:           recorder,
-		workqueue:          workqueue.NewRateLimitingQueue(workqueue.DefaultControllerRateLimiter()),
+		workqueue:          workqueue.NewNamedRateLimitingQueue(workqueue.DefaultControllerRateLimiter(), "janitor_controller_installationtargets"),
```

Comment:
Mind dropping a line why we favour workqueue limiter to shipperworkqueue here?

Reply:
Actually, just my mistake. Good catch ^_^